ui/app/common/obs/views/showObservation.html

```diff
@@ -13,7 +13,7 @@
                        'value-text-only': !(observation.concept.dataType ==='Text' && print)
                        }"
               ng-if="::(observation.isFormElement() && !observation.isImageConcept() && !observation.isVideoConcept())">
-            <span>{{::observation.getDisplayValue() | titleTranslate}} </span>
+            <pre class="left">{{::observation.getDisplayValue() | titleTranslate}}</pre>
```

Comment:
Changing from `span` to `pre` with `class="left"` fixed the whitespace removal issue? Now it shows up exactly as entered? How does this change solve the problem?

Reply:
@gsluthra,
Please go through the doc
https://docs.google.com/document/d/1YURJkEHYZsBuh9_vVHsxaYZqpxKO_PN3kX31AVy4mKc/edit

Reply:
Since this issue was introduced in v0.93 and wasn't there in V0.92, was it also <pre> in the previous version. Just curious.

Reply:
This change was introduced as part of fix for https://bahmni.atlassian.net/browse/BAH-1130. Are we sure this is the right fix and we are not introducing the defect again? @rohit-yawalkar  

```
commit 05ffa8405a11780225c5b8d4a5747a6586a3f2f3
Author: Romain Buisson <rbuisson@users.noreply.github.com>
Date:   Mon Sep 13 14:16:50 2021 +0200

    BAH-1130 | Replace <pre> by <span> to display obs value (#374)

diff --git a/ui/app/common/obs/views/showObservation.html b/ui/app/common/obs/views/showObservation.html
index b465f1ba3..f575f303e 100644
--- a/ui/app/common/obs/views/showObservation.html
+++ b/ui/app/common/obs/views/showObservation.html
@@ -13,7 +13,7 @@
                        'value-text-only': !(observation.concept.dataType ==='Text' && print)
                        }"
               ng-if="::(observation.isFormElement() && !observation.isImageConcept() && !observation.isVideoConcept())">
-            <pre>{{::observation.getDisplayValue() | titleTranslate}}</pre>
+            <span>{{::observation.getDisplayValue() | titleTranslate}} </span>
             <span class="label-add-on">{{::observation.concept.units}}</span>
         </span>
         <a  ng-if="::(observation.isImageConcept() && !observation.hasPDFAsValue() && !print)" class="img-concept">
```

Reply:
@N0-man,
Hey Nouman,
In our commit we have converted span(tag) to pre(tag)
https://github.com/Bahmni/openmrs-module-bahmniapps/pull/390
Below is the code behaviour https://docs.google.com/document/d/1YURJkEHYZsBuh9_vVHsxaYZqpxKO_PN3kX31AVy4mKc/edit
Please check once

And regarding this card, https://bahmni.atlassian.net/browse/BAH-1130, the commit which is made on this https://github.com/Bahmni/openmrs-module-bahmniapps/pull/337/files
is keeping pre tag as it is, the only difference is `<pre class="left">` is changed to `<pre>`, as class left will not make any behaviour change
We can remove that as well, we kept it because it was the same for v0.92, please check below’s commit
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/release-0.92/ui/app/common/obs/views/showObservation.html (line no 16)

Reply:
@rohit-yawalkar based on our discussion it looks like you have already tested this and its working as expected. merging the PR